### ISSUE TITLE
Auto scroll or show only the last values

### ISSUE THREAD

**Issue by roboserg** [mminer/consolation#20]
What the titles says. If I log the speed, I dont need to see old values, only the current ones. Also auto scroll would be cool

**Comment by mminer** [comment]
Hmm, autoscroll is *supposed* to work. Perhaps it's broken.

Regarding not seeing old values, that's an interesting idea. Perhaps we could provide an option to collapse logs fired from the same line of code.

**Comment by mminer** [comment]
I just checked the autoscroll behaviour and it appears to work when you're scrolled to the bottom. @roboserg, is this something that doesn't work for you? What version of Unity are you using?

**Comment by jgalmeidah** [comment]
I am using Unity 2018.4.16 on Andriod.  What I see is "scroll lock" : even if scrolling to the bottom or the middle the current view is retained (until updated).  I tried several times scrolling to the bottom, but is still behaves that way.
Maybe it would be better to have a "scroll lock" button?

**Comment by mminer** [comment]
Agreed, it'd maybe be better to have more explicit controls to set the scroll behaviour.
